netty/src/main/java/io/grpc/netty/NettyServerProvider.java

```diff
@@ -38,5 +39,15 @@ protected int priority() {
   protected NettyServerBuilder builderForPort(int port) {
     return NettyServerBuilder.forPort(port);
   }
+
+  @Override
+  protected NewServerBuilderResult newServerBuilderForPort(int port, ServerCredentials creds) {
+    ProtocolNegotiators.FromServerCredentialsResult result = ProtocolNegotiators.from(creds);
+    if (result.error != null) {
+      return NewServerBuilderResult.error(result.error);
+    }
+    return NewServerBuilderResult.serverBuilder(
+        new NettyServerBuilder(new InetSocketAddress(port), result.negotiator));
```

Comment:
So we have already made changes to use `InetSocketAddress` instead of just `port`. This would make it easy to expose IP address binding in external APIs. So if that is planned for a future PR that sounds good.